packages/editor/src/components/post-title/index.native.js

```diff
@@ -70,7 +72,11 @@ class PostTitle extends Component {
 		const borderColor = this.state.isSelected ? focusedBorderColor : 'transparent';
 
 		return (
-			<View style={ [ styles.titleContainer, borderStyle, { borderColor } ] }>
+			<View
+				style={ [ styles.titleContainer, borderStyle, { borderColor } ] }
+				accessible={ ! this.state.isSelected }
+				accessibilityLabel={ __( 'Post title' ) + '. ' + ( isEmpty( title ) ? __( 'Empty' ) : title ) }
```

Comment:
Just a quick note about this:

Although most languages use a period (`.`) to end a sentence, some don't, and we might think that it would be more correct to use `sprintf` in this case to concatenate the 2 sentences.
```
sprintf( __( 'Post title. %s' ), isEmpty( title ) ? __( 'Empty' ) : title )
```
However I don't think it's worth it at this point. it's not very clear for translators what the `Post title. %s` string means (even with a comment). Moreover `Post title` will probably be reused at other places so it makes more sense to translate that only.

Lastly, this sentence is built to be heard not read by the user.

Reply:
Updating this according to what we discussed

Reply:
Good catch! We can refer to this one for the future accessibility labels

Reply:
+1! let's use this for the next labels.